src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/EETypeCreator.cs

```diff
@@ -184,6 +187,9 @@ internal unsafe class EETypeCreator
                     flags = pTemplateEEType->Flags;
                     isArray = pTemplateEEType->IsArray;
                     isGeneric = pTemplateEEType->IsGeneric;
+#if EETYPE_MODULE_MANAGER
+                    moduleManager = pTemplateEEType->PointerToTypeManager;
```

Comment:
>moduleManager = pTemplateEEType->PointerToTypeManager [](start = 20, length = 53)

rename moduleManager to typeManager (and all the #if names to something with TYPE_MANAGER maybe). It's confusing otherwise